creator-node/src/utils.js

```diff
@@ -104,11 +113,9 @@ async function rehydrateIpfsFromFsIfNecessary (req, multihash, storagePath, file
   }
 
   let rehydrateNecessary = false
+  // Race condition to either let 1s pass or fetch a single byte from ipfs
```

Comment:
did you meant to leave this comment in? i think this was relevant when Promise.race() was used

Reply:
whoops, will remove!